presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -141,7 +147,7 @@ public void run()
                 command.runCommand();
             }
             catch (Exception e) {
-                // TODO: Add to logger
+                LOGGER.error("Encountered exception", e);
```

Comment:
Yes. Whenever I throw PrestoManagerException, I add a description reg that error.